### PR TITLE
Improve visibility for access_key token exception in get_token()

### PR DESCRIPTION
**Pull request by MeganerdDev**
# Closes: #197 

## What's Changed

Adds a try/except block for slicing access_token from response
Adds error logger message to stdout on exception slicing get_token() response.json()["access_key"]
Adds debug logger message to stdout on get_token() that prints json response data

## TODO

- [x] Explanation of Change(s)
- [x] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/chatops/en/latest/dev/dev_contributing/#creating-changelog-fragments))
- [ ] Attached Screenshots, Payload Example
- [ ] Unit, Integration Tests
- [ ] Documentation Updates (when adding/changing features)
- [ ] Outline Remaining Work, Constraints from Design

### CODE REVIEW

File: nautobot_chatops/dispatchers/ms_teams.py [hunk 1]
```diff
@@ -63,7 +63,12 @@ def get_token():
             },
             timeout=15,
         )
-        token = response.json()["access_token"]
+        logger.debug(f"DEBUG: get_token() response {response.json()}")
+        try:
+            token = response.json()["access_token"]
+        except(KeyError) :
+            logger.error("ERROR: get_token() response is missing access_token key")
+            raise KeyError("get_token() response is missing access_token key")
```
[the remark this answers is not in the document]
Comment (MeganerdDev) [reply nautobot_chatops/dispatchers/ms_teams.py:71]: Good point

[the remark this answers is not in the document]
Comment (MeganerdDev) [reply nautobot_chatops/dispatchers/ms_teams.py]: Updated to:
"get_token() response is missing access_token key, which indicates an error communicating with MS Teams. Check the app_id and app_secret."

**Comment by jvanderaa** [comment]
@MeganerdDev I think this makes sense, but we do need to get the tests passing.

**Review by smk4664 — APPROVED** [review]
Thanks for adding this debugging, this will be very helpful in troubleshooting MS Teams issues.

**Pull request merged**